packages/examples/cvat/exchange-oracle/pyproject.toml

```diff
@@ -28,19 +28,104 @@ pyinstrument = "^4.6.2"
 
 
 [tool.poetry.group.dev.dependencies]
-black = "^23.1.0"
 pre-commit = "^3.0.4"
-isort = "^5.12.0"
+ruff = "^0.6.0"
 
-[tool.isort]
-profile = "black"
-forced_separate = ["tests"]
-line_length = 100
-skip_gitignore = true # align tool behavior with Black
-
-[tool.black]
+[tool.ruff]
 line-length = 100
-target-version = ['py310']
+target-version = "py310"
+
+
+[tool.ruff.lint]
+select = ["ALL"]
+unfixable = [
+    "RUF005",  # messes up concantenation with numpy structures
+]
+ignore = [
+    "ANN101",  # method args annotations (mypy will take care of that)
+    "ANN001",  # |
+    "ANN202",  # |
+    "ANN201",  # |
+    "ANN401",  # |
+    "ANN102",  # |
+    "RUF001",  # Allow cyrillic letters in comments
+    "COM812",  # Trailing comas are handled by ruff format
```

Comment:
If it's handled by ruff formatting, why it's ignored?

Reply:
Added other conflicting rules to that list and added clarification as well. 

See https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules.